nwb_conversion_tools/datainterfaces/sipickledatainterfaces.py

```diff
@@ -1,18 +1,18 @@
 """Authors: Alessio Buccino."""
 import spikeextractors as se
-from pynwb import NWBFile
-from pynwb.device import Device
-from pynwb.ecephys import ElectrodeGroup, ElectricalSeries, SpikeEventSeries
 
-from ..basedatainterface import BaseDataInterface
-from ..utils import get_base_schema, get_schema_from_method_signature, \
-    get_schema_from_hdmf_class
+from ..baserecordingextractorinterface import BaseRecordingExtractorInterface
+from ..basesortingextractorinterface import BaseSortingExtractorInterface
 
 
-class SIPickleRecordingExtractorInterface(BaseDataInterface):
+class SIPickleRecordingExtractorInterface(BaseRecordingExtractorInterface):
+    """Primary interface for reading and converting SpikeInterface objects through Pickle files."""
+
+    RX = None
```

Comment:
Why is this needed?

Reply:
With this version that inherits from the BaseRecordingExtractorInterface, I assume it would only be proper to contain the RX class object common to all RecordingExtractorInterfaces, with the None value indicating it is not by default any particular class from SpikeExtractor.

That said, since we're not relying on the class for instantiation of the actual instance of the extractor in the __init__, I suppose it wouldn't actually get used unless something downstream wanted to test whether this interface has an RX object that is None or something.

We could also update the status of RX perhaps after loading the recording object during the __init__ here, I think.

Reply:
This class would already inherit the `RX = None` from the `BaseRecordingExtractorInterface` so this line is redundant

Reply:
Oh, that's true. Fixing that now